modules/broker-core/src/main/java/org/wso2/broker/core/Metadata.java

```diff
@@ -122,173 +108,67 @@ public long getContentLength() {
     }
 
     public void addOwnedQueue(String queueName) {
-        queueList.add(queueName);
-    }
-
-    public List<String> getOwnedQueues() {
-        return queueList;
-    }
-
-    public FieldValue getHeader(ShortString propertyName) {
-        parseHeaders();
-        return headers.getValue(propertyName);
+        queueSet.add(queueName);
     }
 
-    public ByteBuf getRawMetadata() {
-        return rawMetadata;
+    public boolean hasAttachedQueues() {
```

Comment:
Shall we use a single word denote the attached queues (attached or owned)?

Reply:
Used the word attached in all the places